package.json

```diff
@@ -2,8 +2,11 @@
   "name": "root",
   "private": true,
   "scripts": {
+    "prettier:check": "npm run prettier -- --check",
     "prettier": "prettier --write 'newdle/client/src/**/*.{js,jsx,json,scss,css}'",
+    "isort:check": "npm run isort -- --check-only",
```

Comment:
I'd include `--diff` so it prints the diff of the failure to the CI output

(same for the other commands; it's nicer if you can make them print diffs in case they'd change something)

Reply:
Agree!